### ISSUE TITLE
Debian: zfs-initramfs should depend on busybox

### ISSUE THREAD

**Issue by dsanader** [zfsonlinux/pkg-zfs#158]
The ZFS initrd scripts rely on various binaries provided by BusyBox (eg. grep), but the zfs-initramfs package doesn't depend on it. This leads to avoidable fallbacks to the rescue shell when busybox is not installed :

```
Begin: Running /scripts/local-top ... Begin: Running /scripts/local-premount ... Scanning for Btrfs filesystems
/init: 382: /init: grep: not found
Begin: Sleeping for 0 seconds... ... Begin: Sleeping for 0 seconds... ... /init: 453: /init: grep: not found
/init: 481: /init: sed: not found
Begin: Importing ZFS root pool 'tank' ... /init: 114: [: yes: unexpected operator
```

**Comment by FransUrbo** [comment]
Nice catch! I've fixed this in version <code>0.6.4-20-16421a</code> for both the Wheezy and Jessie packages. As soon as we create a new ZoL version, I'll update the released packages as well.

**Comment by dsanader** [comment]
Thanks for the quick review. Regards.

**Comment by FransUrbo** [comment]
I've now fixed this in the <code>0.6.4-1.2-1</code> packages as well, so I consider this fully fixed and closed.
